tests/author.test.js

```diff
@@ -0,0 +1,132 @@
+const { expect } = require("chai");
+const request = require("supertest");
+const { Author } = require("../src/models");
+const app = require("../src/app");
+
+describe("/authors", () => {
+  before(async () => Author.sequelize.sync());
+
+  beforeEach(async () => {
+    await Author.destroy({ where: {} });
+  });
+  describe("with no authors on database", () => {
+    describe("POST /authors", () => {
+      it("creates a new author in the database", async () => {
+        const response = await request(app).post("/authors").send({
+          author: "Brian Jacques",
+        });
+
```

Comment:
remove clear line and add it after line 21 to group execution and assertions together like you have done in the tests at the end of this file - consistency